### PR TITLE
Use event name as prefix for Algolia indices

### PR DESCRIPTION
**Pull request by rock3r**
## Problem

Our indices on Algolia are all on the same project, so we cannot currently host more than one conference there.

## Solution

As their support suggested, we should prefix our indices with a unique conference name, to allow hosting multiple conferences on the same Algolia instance.

The conference name comes from the Firebase Functions environment configurations.

### Test(s) added 

Nope

### Paired with 

Nobody

### CODE REVIEW

File: functions/src/search/index-contents.ts [hunk 1]
```diff
@@ -13,13 +13,18 @@ export const indexContent =
...
         ]).then(() => {
             response.status(200).send('Yay!')
         }).catch(error => {
             console.error(error)
             response.status(500).send('Nay!')
         })
     }
+
+const replaceNonWordCharsWithUnderscores = (original: string): string => {
+    return original.replace(/[^\w]/, '_')
```
Comment (fourlastor) [inline functions/src/search/index-contents.ts]: I'd throw if this is empty here

Comment (rock3r) [reply functions/src/search/index-contents.ts]: oh yeah I wanted to do it then I forgot LOL

Comment (rock3r) [reply functions/src/search/index-contents.ts]: I have done a method in the other PR, so once that's merged I'll pull it in here too

Comment (rock3r) [reply functions/src/search/index-contents.ts]: 5b2c5c00c4b179e7ab1b84ff120f51030a2836a7

**Pull request merged**